### DISCUSSION TITLE
Link that forces reload?

### DISCUSSION THREAD

**Discussion by therealgilles** [gatsbyjs/gatsby#29543] [Help]
Is there a way to create a link that forces page reload? This is useful at least during development.

**Comment by KyleAMathews** [comment]
You can use a standard `<a>` tag

**Reply by therealgilles** [reply]
That did not work for me, at least in development, have not tried in production. The following code works for me:

```JS
   <Link to="/" onClick={() => (window.location = "/")}>... </Link>
```

**Reply by KyleAMathews** [reply]
Hmm perhaps you have the catch link plugin installed? Normally that works.

**Reply by therealgilles** [reply]
I do not have that plugin installed.

**Reply by therealgilles** [reply]
Oh wait, I see it was automatically installed by gatsby-source-wordpress.

**Reply by therealgilles** [reply]
Is this documented somewhere? If not, it would be nice if it were.

**Reply by therealgilles** [reply]
@KyleAMathews: Maybe I should have phrased that a bit differently. Is there a place I could make a PR to document it? and is there a way to disable the plugin? I saw it's possible to add exclusions but I am not sure how to do that when the plugin is specified in the gatsby-config.js file inside the gatsby-source-wordpress plugin. I'm also curious as to why it was decided to include it there.

**Reply by KyleAMathews** [reply]
It should be documented in the source plugin's readme so PRing that there would be great!

I suspect they added it as WordPress content has a lot of links inline the content so those would work client-side as well.

**Reply by therealgilles** [reply]
Any ideas about my other questions? i.e. is there a way to disable or configure the plugin? Oh and is my workaround the only possible alternative with the plugin enabled?
